prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,45 @@
+from taxi import Taxi
+from silver_service_taxi import SilverServiceTaxi
+
+MENU = "q)uit, c)hoose taxi, d)rive"
+
+def main():
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+    current_taxi = None
+    total_cost = 0
+    print("Let's drive!")
+    print(MENU)
+    choice = input(">>>").upper()
+    while choice != 'Q':
+        if choice == 'C':
+            print("Taxis available: ")
+            for index in range(len(taxis)):
+                print(f"{index} - {taxis[index]}")
+            choose_taxi = int(input("Choose taxi: "))
+            if choose_taxi < 0 or choose_taxi > len(taxis) - 1:
+                print("Invalid taxi choice")
+            current_taxi = taxis[index]
```

Comment:
I think this line should be under else, if not then even if the input is invalid, the current taxi has a value other than false or none which would be an error when picking drive choice later